skr/config/manager/kustomization.yaml

```
@@ -16,5 +16,5 @@ images:
 - name: controller-manager
   newName: docker.io/jaythedevil666/kyma-watcher
 - name: kyma-watcher
-  newName: jaythedevil666/kyma-watcher
+  newName: k3d-registry.localhost:4000/kyma-watcher
```

please add a placeholder here, it will fail and the user will know what's missing
```suggestion
  newName: <your-own-skr-watcher-image>
```